infra/gcb/templates/src/build-status/build-status.html

```diff
@@ -216,8 +322,25 @@
         let timezone = ds.substring(ds.indexOf("("));
         return date.toLocaleString() + " " + timezone;
       }
-    };
 
-    window.customElements.define(BuildStatus.is, BuildStatus);
+      pad(n) {
```

Comment:
nit: format this properly. (i.e. remove the extra {} and the extraneous newline)